frontend/src/pages/budgetLines/PreviewTable.scss

```diff
@@ -3,3 +3,8 @@
 .usa-table th {
     padding: 0.5rem 0.5rem;
 }
+
+.underReview {
```

Comment:
maybe we can add this color to our global styles in `/sass/uswds/styles.scss` as suggested name:
`--data-viz-secondary-23: #8B4687`